src/InteractiveWindow/Editor/Commands/ResetCommand.cs

```diff
@@ -51,53 +52,66 @@ public override string CommandLine
 
         public override Task<ExecutionResult> Execute(IInteractiveWindow window, string arguments)
         {
-            int noConfigStart, noConfigEnd;
-            if (!TryParseArguments(arguments, out noConfigStart, out noConfigEnd))
+            bool initialize;
+            if (!TryParseArguments(arguments, out initialize))
             {
                 ReportInvalidArguments(window);
                 return ExecutionResult.Failed;
             }
 
-            return window.Operations.ResetAsync(initialize: noConfigStart > -1);
-        }
-
-        internal static string BuildCommandLine(bool initialize)
-        {
-            string result = CommandName;
-            return initialize ? result : result + " " + NoConfigParameterName;
+            return window.Operations.ResetAsync(initialize);
         }
 
         public override IEnumerable<ClassificationSpan> ClassifyArguments(ITextSnapshot snapshot, Span argumentsSpan, Span spanToClassify)
         {
             string arguments = snapshot.GetText(argumentsSpan);
+            int argumentsStart = argumentsSpan.Start;
+            foreach (var pos in GetNoConfigPositions(arguments))
+            {
+                var snapshotSpan = new SnapshotSpan(snapshot, new Span(argumentsStart + pos, NoConfigParameterNameLength));
+                yield return new ClassificationSpan(snapshotSpan, _registry.Keyword);
+            }
+        }
 
-            int noConfigStart, noConfigEnd;
-            if (TryParseArguments(arguments, out noConfigStart, out noConfigEnd))
+        /// <remarks>
+        /// Internal for testing.
+        /// </remarks>
+        internal static IEnumerable<int> GetNoConfigPositions(string arguments)
+        {
```

Comment:
I wanted to use ```Split``` but it doesn't retain position information (i.e. the width of the whitespace is discarded).